test/unit/support/shared/virtualbox_context.rb

```diff
@@ -28,6 +28,11 @@ def subprocess_result(options={})
     allow(subprocess).to receive(:execute).
       with("VBoxManage", "showvminfo", kind_of(String), kind_of(Hash)).
       and_return(subprocess_result(exit_code: 0))
+
+    # apparently this is also used in docker tests
```

Comment:
Interesting. I removed this and ran the tests myself and didn't receive any error at all requiring this. Do you have an example of the error you were receiving without this?

Reply:
Yeah, see https://travis-ci.org/hashicorp/vagrant/builds/538053443.

Reply:
It’s possible that the discrepancy is because my commit is based on v2.2.4 and Travis ran on a merge of my commit with master.

Reply:
Hmm, strange. Well I'll see if it happens to me when I open a new PR with the rest of the tests. I've tried a couple of times locally and it was totally fine without it :thinking: Thanks for the info!

Reply:
The discrepancy turns out to be because the `usable?` method was no longer called after you changed it from a class method to an instance method.